softwarereview_editor.Rmd

```diff
@@ -8,74 +8,72 @@ aliases:
 The Editor-in-Chief (EiC) role is rotated (generally quarterly) amongst experienced members of our editorial board.
-Information on current and recent editors and their activities can be viewed on our editorial dashboard at [dashboard.ropensci.org](https://dashboard.ropensci.org).
+Information on current status of all editorial team members is presented on our [*Editorial Dashboard*](https://dashboard.ropensci.org).
 
 This chapter presents the responsibilities [of the Editor-in-Chief](#eicchecklist), and of [handling editors in charge of submissions](#editorchecklist).
 It also describes [how to respond to an out-of-scope submission](#outofscoperesponse), and provides guidance on [answering reviewers' questions](#reviewersupport).
 
 If you're a guest editor, thanks for helping! Please contact the editor who invited you to handle a submission for any question you might have.
 ```
 
-> Always assume participants in the software review system (fellow editors, submitters, reviewers) are doing their best, and communicate gracefully accordingly, especially when inquiring why a thing is delayed.
+This chapter is structured to reflect the typical progression of an rOpenSci software review.
+All submissions are initially considered by the Editor-in-Chief, who makes the initial decision on whether a package is in scope.
+If so, a handling editor is assigned who is then responsible for guiding the actual review process.
+
+Please be aware that all people involved in the review process are volunteers who are freely and willingly devoting their own time.
+Always assume all participants in the software review system (fellow editors, submitters, reviewers) are doing their best, and communicate gracefully at all times.
 
 ## EiC Responsibilities {#eicchecklist}
 
 Rotating Editors-in-Chief (EiCs) generally serve for 3 months or a time agreed to by all members of the editorial board.
-The EiC is entitled to taking scope and overlap decisions as independently as possible (but can still request help and advice).
-Information on current status of all editorial team members is presented on our [*Editorial Dashboard*](https://dashboard.ropensci.org).
-The EiC is responsible for the following tasks:
-
-- On assuming EiC rotation, reviewing the status of current open reviews as detailed on the [*Dashboard* page](https://dashboard.ropensci.org/reviews.html), and issuing reminders to other editors or package authors as needed. See [the following sub-section for more details](#eic-dashboard)
-
-- Watching all new issues posted to the software-review repo, for which the EiC must either subscribe to repo notifications on GitHub, or watch the `#software-peer-review-feed` channel on Slack.
+The EiC is responsible for [general management](#eic-general) of the review process, and for the [initial stages](#eic-submission) of all submissions.
 
-- Tagging each new full submission with ` 0/editorial-team-prep`
+### General EiC duties {#eic-general}
```

Comment:
on the dashboard page, how do you

- know when it's last been updated
- get a link to the workflow building it to investigate status if need be?